planner/core/src/main/java/eu/seaclouds/platform/planner/core/Planner.java

```
@@ -154,7 +161,7 @@ public String rePlan() throws ParsingException, IOException {
         //Optimize
         String mmOutput = "";
         try {
-            mmOutput = generatMMOutput(matchingOfferings, offerings);
+            mmOutput = generateMMOutput(matchingOfferings, offerings);
         } catch (Exception e) {
             //What should we return in case of failure?
             e.printStackTrace();
```

don't ignore exceptions